README.md

```diff
@@ -5,4 +5,72 @@
 + You have access to the database
 + Useful for sandboxing, testing associations, checking app.
 + Prewritten code is a sinatra application with a few associations
-+ Use activerecord methods in tux (.all, .new, .save, .create, .first, etc)
\ No newline at end of file
++ Use activerecord methods in tux (.all, .new, .save, .create, .first, etc)
+
+## What Is Tux
+
+Tux is an incredible Ruby gem that let's you access your database and perform all CRUD operations on it through the terminal. It also loads a full environment in the console that allows you to see all routes and views. Primarily, you'll use Tux to make sure your database is set up properly, play around with Ruby objects, and make sure your ActiveRecord associations are working properly.
```

Comment:
I would be a bit show-yer on why tux is amazing. People won't fully understand it unless they have to feel the pain. So maybe something like "in this repo there is a simple sinatra app with a model and a method called blah. I want to test that the blah method works. So I type irb and require the file, which then requires me to require all this other stuff. (show them the errors and all that). That's annoying which is why Tux exists"